miner/worker.go

```diff
@@ -1080,6 +1082,65 @@ func (w *worker) fillTransactions(interruptCh chan int32, env *environment, stop
 			localBlobTxs[account] = txs
 		}
 	}
+
+	if bidTxs != nil {
+		for acc, txs := range localPlainTxs {
+			for i := len(txs) - 1; i >= 0; i-- {
+				if bidTxs.Contains(txs[i].Hash) {
+					txs = txs[i+1:]
+					break
```

Comment:
we get a problem, shall we break here?

You know it is possible that account A can have two tx1 and tx2 in the `bidTxs`, if the logic here match the tx1, then it will still have the tx with same nonce with tx2 in the pool, so I think we may should not use txhash to detect conflict, but use the nonce of an account to detect the conflict.